aldryn_people/boilerplates/bootstrap3/templates/aldryn_people/group_detail.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
     <div class="aldryn aldryn-people aldryn-group-detail">
         <h1>{% render_model group 'name' %}</h1>
-        {% for person in group.people.all %}
+        {% for person in group.people.all|dictsort:"name" %}
```

Comment:
indeed. if it's too complicated to get it sorted from the db, though, i think it's ok to sort in memory and optimize later. usually lists of people wont be very long anyway.